ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -0,0 +1,456 @@
+import static org.hamcrest.Matchers.*;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.*;
+import java.security.SignatureException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static com.spectralogic.ds3client.helpers.Ds3ClientHelpers.*;
+import static com.spectralogic.ds3client.integration.Util.*;
+import static org.junit.Assert.*;
+
+public class PutJobManagement_Test {
+
+    private static Ds3Client client;
+    private static final String TEST_ENV_NAME = "PutJobManagement_Test";
+    private static TempStorageIds envStorageIds;
+
+    @BeforeClass
+    public static void startup() throws IOException, SignatureException {
+        client = Util.fromEnv();
+        final UUID dataPolicyId = TempStorageUtil.setupDataPolicy(TEST_ENV_NAME, true, ChecksumType.Type.MD5, client);
+        envStorageIds = TempStorageUtil.setup(TEST_ENV_NAME, dataPolicyId, client);
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException, SignatureException {
+        TempStorageUtil.teardown(TEST_ENV_NAME, envStorageIds, client);
+        client.close();
+    }
+
+    private void checkTimeOut(final long startTime, final int testTimeOutSeconds){
+        assertTrue((System.nanoTime() - startTime)/1000000000 <= testTimeOutSeconds);
+        assertThat((System.nanoTime() - startTime)/1000000000, lessThan((long) testTimeOutSeconds));
+    }
+
+    @Test
+    public void modifyJobPriority() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_priority";
+        try {
+            Ds3ClientHelpers.wrap(client).ensureBucketExists(bucketName);
+            
+            final List<Ds3Object> objects = Lists.newArrayList( new Ds3Object("test", 2));
+
+            final WriteJobOptions jobOptions = WriteJobOptions.create().withPriority(Priority.LOW);
+
+            final Ds3ClientHelpers.Job job =
+                    wrap(client).startWriteJob(bucketName, objects, jobOptions);
+
+            client.modifyJobSpectraS3(new ModifyJobSpectraS3Request(job.getJobId())
+                    .withPriority(Priority.HIGH));
+
+            final GetJobSpectraS3Response response = client
+                    .getJobSpectraS3(new GetJobSpectraS3Request(job.getJobId()));
+
+            assertThat(response.getMasterObjectListResult().getPriority(), is(Priority.HIGH));
+
+        } finally {
+            deleteAllContents(client, bucketName);
+        }
+    }
+
+    @Test
+    public void modifyJobName() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_name";
+        try {
+            Ds3ClientHelpers.wrap(client).ensureBucketExists(bucketName);
```

Comment:
Well, the wrap() part is . . . should I be creating a Ds3ClientHelpers helpers = Ds3ClientHelpers.wrap(client); and calling my functions from helpers.xxx()?  If so, maybe it belongs in @before along with the bucket creation, since they're called in every test?